src/main/resources/hudson/maven/Messages.properties

```diff
@@ -55,7 +55,7 @@ MavenProbeAction.DisplayName=Monitor Maven Process
 MavenProcessFactory.ClassWorldsNotFound=No classworlds*.jar found in {0} -- Is this a valid maven directory?
 
 MavenRedeployer.DisplayName=Deploy to Maven repository
-MavenVersionCallable.MavenHomeDoesntExist=Maven Home {0} doesn\u2019t exist
+MavenVersionCallable.MavenHomeDoesntExist=Maven Home {0} doesn't exist
```

Comment:
@demonfiddler  Sure, thanks. I just hadn't seen the commit comment I quoted yet. What I wonder is how and by whom was that character deprecated?